src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlSerializableServices.cs

```diff
@@ -52,7 +52,7 @@ private static bool IsValidAttribute(XmlReader xmlReader)
                                    xmlReader.LocalName != "xmlns";
         }
 
-        public static void WriteNodes(XmlWriter xmlWriter, XmlNode[] nodes)
+        public static void WriteNodes(XmlWriter xmlWriter, XmlNode[]? nodes)
```

Comment:
```suggestion
        public static void WriteNodes(XmlWriter xmlWriter, XmlNode?[]? nodes)
```
Seems like code will ignore null elements.

Reply:
Nice catch! Fixed.